docs/collections/returning-collection-data.md

```diff
@@ -43,28 +43,23 @@ From source,
 </Tabs>
 
 
-### Cursor API
+### Cursor
 
-For the Cursor API solution, create a cursor to receive the results from your request. When calling the `query` method request options are provided as key/value pairs. It is important to set the `batchSize` value to 1000 and the `stream` value to true. This will permit the results from the query to be returned as a stream, allowing us to return subsequent batches of data.
+We can create a cursor to receive the results from our request. A cursor details where we left off when reading a stream of responses from API, so we can continue to read the batch of required data. When calling the `query` method request options are provided as key/value pairs. For a detailed explanation of options please visit [Api Reference](https://macrometa.com/docs/api#/operations/createQueryCursor). It is important to set the `batchSize` to required value (code ex. 1000) and the `stream` value to true. This will permit the results from the query to be returned as a stream, allowing us to return subsequent batches of data. In response, the entire result set is loaded into the cursor. (code ex. `cursor`) Therefore we must be mindful of client-side memory capacity when running queries that can potentially return a large result set.
```

Comment:
```suggestion
We can create a cursor to receive the results from our request. A cursor details where we left off when reading a stream of responses from API, so we can continue to read the batch of required data. When calling the `query` method request options are provided as key/value pairs. For a detailed explanation of options visit the [API reference](https://macrometa.com/docs/api#/operations/createQueryCursor). Set the `batchSize` to the required value (code ex. 1000) and the `stream` value to true. This will permit the results from the query to be returned as a stream, allowing us to return subsequent batches of data. In response, the entire result set is loaded into the cursor (code ex. `cursor`). Therefore we must be mindful of client-side memory capacity when running queries that can potentially return a large result set.
```

A few minor grammatical/concision suggestions here.  Also, no need to say "please" in technical docs.

Otherwise looks good!

Reply:
Thank you @jamesm-macrometa! 👍